api/src/tests/move/pack_counter/sources/counter.move

```diff
@@ -0,0 +1,26 @@
+module addr::counter {
```

Comment:
license headers?

Reply:
Do we need them for test sources as well?